tiatoolbox/tools/registration/wsi_registration.py

```diff
@@ -3,6 +3,8 @@
 
 import cv2
 import numpy as np
+import scipy.ndimage as ndi
+import SimpleITK as Sitk
```

Comment:
```suggestion
import SimpleITK as sitk
```

Reply:
Fixed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/488/commits/025c1da7e0ad34558ce9c17edf37196889be6ed7